### PR TITLE
[RFR] Inist rdf exporters

### PR DESCRIPTION
**Pull request by touv**

### CODE REVIEW

**Comment by touv** [comment]
@parmentf RDF export is available but it should be improved later with :
- grouped fields
- completed fields
- Dereferencing HTTP URIs
- and probably more !

**Review by djhi — DISMISSED** [review]
Looks like you're missing an npm package: `n3`

File: docker-compose.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ server:
         - mongo
     ports:
         - 3000:3000
+        - 8080:8080
```
Comment (djhi) [inline docker-compose.yml]: why ?

File: package.json [hunk 2]
```diff
@@ -43,14 +43,15 @@
...
-    "ezs-basics": "2.2.1",
+    "ezs-basics": "2.2.5",
     "file-loader": "0.9.0",
     "html-webpack-plugin": "2.26.0",
     "inist-ark": "2.0.0",
     "isomorphic-fetch": "2.2.1",
     "jbj": "7.1.4",
     "json-csv": "1.3.0",
     "json-loader": "0.5.4",
+    "jsonld": "^0.4.11",
```
Comment (djhi) [inline package.json]: please fix the versions by not using neither `^` nor `~`

**Review by djhi — DISMISSED** [review]

File: package.json (hunk 2 above)
Comment (djhi) [reply package.json]: You can configure your npm to always save exact versions: [article](http://c2fo.io/node/npm/javascript/2016/06/03/protecting-your-product-with-npm-save-exact/)

Comment (touv) [reply package.json]: ok, so we can use a per-project config file - /path/to/my/project/.npmrc 
is it a good idea ?

**Review by djhi — APPROVED** [review]

**Pull request merged**